recipes-tag/databroker/meta.yaml

```diff
@@ -21,7 +25,6 @@ requirements:
     - boltons
     - cytoolz
     - doct
-    - glueviz
```

Comment:
Please don't delete these optional dependencies from the recipe. They are optional for the package but required on the floor. (We need pymongo, especially.)